narwhals/_arrow/expr.py

```diff
@@ -135,3 +145,22 @@ def to_string(self, format: str) -> ArrowExpr:  # noqa: A002
         return reuse_series_namespace_implementation(
             self._expr, "dt", "to_string", format
         )
+
+
+class ArrowExprStringNamespace:
+    def __init__(self, expr: ArrowExpr) -> None:
+        self._expr = expr
+
+    def to_uppercase(self) -> ArrowExpr:
+        return reuse_series_namespace_implementation(
+            self._expr,
+            "str",
+            "to_uppercase",
+        )
+
+    def to_lowercase(self) -> ArrowExpr:
+        return reuse_series_namespace_implementation(
+            self._expr,
+            "str",
+            "to_lowercase",
+        )
```

Comment:
this looks irrelevant to your PR, I think it might have crept through in a merge conflict? could you clean up your branch so it only contains the changes you intended to make?